css-layout-classes/styles.css

```diff
@@ -0,0 +1,33 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
```

Comment:
You should make this `column-full` and use it instead of the `row-column-full` class you have in your first row that doesn't actually have any styling applied to it.